semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -416,31 +443,36 @@ def _render_group(self, semconv, parameters, output):
         self.render_ctx.is_remove_constraint = "remove_constraints" in parameters
         self.render_ctx.group_key = parameters.get("tag")
         self.render_ctx.is_full = "full" in parameters
+        self.render_ctx.is_metric_table = "metric_table" in parameters
 
         if isinstance(semconv, EventSemanticConvention):
```

Comment:
This logically also belongs to the "not metrics" branch.

Reply:
Done 👍